File: 0162-find-peak-element/0162-find-peak-element.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findPeakElement = function(nums) {
   let max =nums[0];
   for(let i=0;i<nums.length;i++){
    if(max < nums[i]){
        max = nums[i];
    }
   } 
   for(let i=0;i<nums.length;i++){
    if(max === nums[i]){
        return i;
    }
}
};

File: 0162-find-peak-element/0162-find-peak-element.py
class Solution(object):
    def findPeakElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        maxs = nums[0]
        for i in range(len(nums)):
            if maxs<nums[i]:
                maxs = nums[i]
        for i in range(len(nums)):
            if maxs ==nums[i]:
                return i